Module.php

```diff
@@ -115,7 +115,10 @@ class Module extends \yii\base\Module implements BootstrapInterface
-
+    /**
+     * @var bool Verify reinitialization
+     */
+    private static $_isInit = false;
```

Comment:
Why static? This will make impossible to instantiate module more than once.

Reply:
I added a test. He must explain everything.

Reply:
Not really. You're not testing whether debug module works after such initialization. You expect "not found" exception, but it is unclear why. You are also using a hack to make reinitialization even possible. 

If problem is only in accesing view on asset panel initilization, why not just wrap `Yii::$app->view->assetManager` into `isset()`?

```php
     public function isEnabled() 
     { 
         try { 
             return isset(Yii::$app->view->assetManager); 
         } catch (InvalidConfigException $exception) { 
             return false; 
         } 
    } 
```

Reply:
`isset` will always return` false`

Reply:
> You are also using a hack to make reinitialization even possible.

I mimic the conditions for the first run, if the test can be run separately, you can exclude

Reply:
> isset will always returnfalse

Are you sure about that?

https://github.com/yiisoft/yii2/blob/master/framework/base/Component.php#L221-L227

Reply:
Yes you are right